internal/engine/sqlite/convert.go

```diff
@@ -168,6 +168,12 @@ func (c *cc) convertCreate_virtual_table_fts5(n *parser.Create_virtual_table_stm
 		}
 	}
 
+	stmt.Cols = append(stmt.Cols, &ast.ColumnDef{
```

Comment:
It should be fine to include the fts5 table name in the column list, as SQLite supports this as well
```
sqlite> SELECT key, emails_fts FROM emails_fts limit 1;
┌───────────────────────────────────────────────────┬────────────┐
│                        key                        │ emails_fts │
├───────────────────────────────────────────────────┼────────────┤
│ 1708522984.M224116P268.c18ac5293f65,S=3578,W=3701 │ 5          │
└───────────────────────────────────────────────────┴────────────┘
```